### DISCUSSION TITLE
Using NodeJS 16

### DISCUSSION THREAD

**Discussion by Kamrulhasan12345** [deta/cloud-docs#474] [General] [answered]
I have a question. DiscordJS latest release requires at 16.x.x. And from deta.sh docs, I can see deta supports only nodejs12 and nodejs14. Can I use NodeJS v16 in deta.sh?

**Comment by jnsougata** [comment, marked as answer]
1. You can't run discord websocket client in deta micros
2. You can specify micro runtine (nodejs version) during the creation of micro

**Reply by Kamrulhasan12345** [reply]
So, deta.sh doesn't support any kind of websocket clietns?

**Reply by jnsougata** [reply]
sadly, no...

**Reply by SuperErnD** [reply]
no :(
